mtags/src/main/scala-3/scala/meta/internal/pc/completions/AmmoniteFileCompletions.scala

```diff
@@ -0,0 +1,120 @@
+package scala.meta.internal.pc
+package completions
+
+import java.nio.file.Path
+import java.{util as ju}
+
+import scala.collection.JavaConverters.*
+
+import scala.meta.internal.mtags.MtagsEnrichments.*
+import scala.meta.internal.pc.AutoImports.AutoImport
+import scala.meta.internal.pc.AutoImports.AutoImportsGenerator
+import scala.meta.internal.pc.printer.MetalsPrinter
+import scala.meta.io.AbsolutePath
+import scala.meta.pc.OffsetParams
+import scala.meta.pc.PresentationCompilerConfig
+import scala.meta.pc.PresentationCompilerConfig.OverrideDefFormat
+import scala.meta.pc.SymbolSearch
```

Comment:
unused (I wish Scala3 compiler could have a flag to detect unused imports in the near future 😭 )

```suggestion
import scala.meta.io.AbsolutePath
```